stl/inc/xstring

```diff
@@ -2648,13 +2638,13 @@ public:
 #if _HAS_CXX23
     constexpr basic_string(basic_string&& _Right, const size_type _Roff, const _Alloc& _Al = _Alloc())
         : _Mypair(_One_then_variadic_args_t{}, _Al) { // construct from _Right [_Roff, <end>), potentially move
-        _Move_construct_from_substr(_Right, _Roff, npos, _Al);
+        _Move_construct_from_substr(_Right, _Roff, npos);
     }
 
     constexpr basic_string(
         basic_string&& _Right, const size_type _Roff, const size_type _Count, const _Alloc& _Al = _Alloc())
         : _Mypair(_One_then_variadic_args_t{}, _Al) { // construct from _Right [_Roff, _Roff + _Count), potentially move
-        _Move_construct_from_substr(_Right, _Roff, _Count, _Al);
+        _Move_construct_from_substr(_Right, _Roff, _Count);

```

Comment:
I think a merge was incorrectly resolved here - please double-check that you've merged with `main` and picked up #3178. There are related stray changes to `_Move_construct_from_substr` below.